packages/accessible-react-app/src/css/primarynav.css

```diff
@@ -14,6 +14,7 @@
 }
 
 .ac-primary-nav__list-item > a {
+  /* SR Talking Point - Color contrast needs to be higher. Don't assume all SR users are blind. */
```

Comment:
How about something like, "/* SR Talking Point - Color contrast needs to be higher for users with color blindness, read more about guideline 1.4.3 at: https://www.w3.org/WAI/WCAG21/quickref/#contrast-minimum */"